src/palette/internal/palette.h

```diff
@@ -161,8 +161,10 @@ class Palette : public QObject
     void showWritingPaletteError(const QString& path) const;
 
     Type guessType() const;
+    static Type guessTypeFrom(const Ms::EngravingItem* e);
```

Comment:
I would suggest removing these definitions from this header. You can declare all the static functions at the top of the palette.cpp file

Reply:
Done, thank you! :slightly_smiling_face: